SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -137,7 +190,7 @@ function Get-Secret
         $Name = $Matches[1]
     }
 
-    $res = Invoke-lpass 'show', '--name', $Name, '--all' -ErrorAction SilentlyContinue
+    $res = Invoke-lpass 'show', '--name', $Name, '--all'
```

Comment:
These are not needed anymore as we deal with everything at the command level.
We only want to disable redirection for login (always) and show (conditionnally) so this is not needed anymore.